teuthology/openstack/__init__.py

```diff
@@ -84,6 +84,7 @@ def __init__(self, name_or_id, info=None):
             errmsg = 'VM creation failed'
             if 'message' in self.info:
                 errmsg = '{}: {}'.format(errmsg, self.info['message'])
+            OpenStack().run("server delete " + self.info.get('id', self.name_or_id))
```

Comment:
This commit is not related to the PR, the "server delete" should not be run directly from the constructor of this class because it does not run "server create", in other words this instruction should be run upper in the stack call.

Reply:
dropped the commit